modules/oe_theme_content_project/oe_theme_content_project.post_update.php

```diff
@@ -54,3 +54,13 @@ function oe_theme_content_project_post_update_00002(): void {
   $display = $storage->createFromStorageRecord($display_values);
   $display->save();
 }
+
+/**
+ * Updates the budget field's 'thousand' separator to space.
+ */
+function oe_theme_content_project_post_update_00003(): void {
+  $view_display = EntityViewDisplay::load('node.oe_project.full');
```

Comment:
We should also update the teaser view mode address formatter, no?

Reply:
Yes, let's do that.